src/components/map/Map.tsx

```
@@ -12,18 +10,25 @@ import { UserInputMeshLayerContainer } from "../userinputmeshlayer/index";
   [35, 139],
   [37, 140],
 ];
 
+/**
+ * Renders an interactive Leaflet map with OpenStreetMap tiles and multiple custom overlay layers.
+ *
+ * The map initializes with predefined bounds and includes layers for debugging, mesh visualization, user input, markers, coordinate popups, and automatic bounding box fitting.
+ *
+ * @returns A React element containing the configured map and its overlays.
+ */
 export function Map() {
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)